plugin/trino-ignite/src/main/java/io/trino/plugin/ignite/IgniteClient.java

```diff
@@ -415,13 +415,6 @@ private String buildCreateSql(SchemaTableName schemaTableName, List<String> colu
         }
         columnDefinitions.add("PRIMARY KEY (" + join(", ", primaryKeys.stream().map(this::quoted).collect(joining(", "))) + ")");
 
-        for (Map.Entry<String, Object> propertyEntry : tableProperties.entrySet()) {
```

Comment:
The commit order is wrong. `tableProperties` is already removed in the 1st commit.

Reply:
@ebyhr thanks. i have fixed it. i verified by running tests on both the commits.